contracts/Redemptions.sol

```diff
@@ -56,6 +58,9 @@ contract Redemptions is AragonApp {
             if (token != ETH) {
                 require(isContract(token), ERROR_TOKEN_NOT_CONTRACT);
             }
+            if (i >= 1) {
+                require(_redeemableTokens[i - 1] < _redeemableTokens[i], ERROR_REDEEMABLE_TOKEN_LIST_MALFORMED);
+            }
```

Comment:
Since we have `redeemableTokenAdded` variable, couldn't we make use of it to check for uniqueness ?

Reply:
Yup can do!